randovania/games/blank/gui/__init__.py

```diff
@@ -0,0 +1 @@
+from .dialog.cosmetic_patches_dialog import BlankCosmeticPatchesDialog
```

Comment:
relative on purpose?

Reply:
Yeah, it was one the first files I did. In retrospect might not be worthwhile.